config/routes.rb

```diff
@@ -2014,6 +2014,9 @@
 
   root :to => 'dashboard#login'
 
+  # Let's serve pictures directly from the DB
+  get '/pictures/:basename' => 'picture#show', :basename => /[\da-zA-Z]+\.[\da-zA-Z]+/
```

Comment:
hmm, anchor characters now allowed in routing requirements.  let's keep above for now.